src/vs/vscode.d.ts

```diff
@@ -8926,6 +8926,19 @@ declare module 'vscode' {
 		customRequest(command: string, args?: any): Thenable<any>;
 	}
 
+	/**
+	 * Options for [starting a debug session](#debug.startDebugging).
+	 */
+	export interface DebugSessionOptions {
+
+		/**
+		 * Controls whether this session should have a separate repl console or share it
+		 * with the parent session. Has no effect for sessions which do not have a parent session.
+		 * Defaults to "separate".
+		 */
+		repl?: 'separate' | 'mergeToParent';
```

Comment:
This should be `debugConsole` not `repl`. We only call it `repl` as an implementation detail, whatever is user facing / api facing we call debug console.

Also `mergeWithParent` sounds better than `mergeToParent`
However I am not a native speaker.
Also would love to get @weinand feddback on naming here.